lib/endpoint/tags/table.js

```diff
@@ -130,8 +130,9 @@ function rowToParameter (i, el) {
     params.type = 'integer[]'
   }
 
-  if (/^array$/.test(params.type)) {
+  if (params.type === 'array') {
     params.type = 'string[]'
+    if (params.name === 'team_ids') params.type = 'number[]'
```

Comment:
could you combine the two lines above with `params.type = params.name === 'team_ids' ? 'number[]' : 'string[]'`? 

And maybe we could make it a bit more future proof by testing with `/_ids$/.test(params.name)` instead of `params.name === 'team_ids'`? What do you think?

Reply:
Yes. One liners always look good.
Great point. :ok: 
There are other id's that may take `array` as input. I'll go change it.